src/app/index.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { createRoot } from 'react-dom/client';
+import { BrowserRouter } from 'react-router-dom';
+import App from 'app/App';
+import { Provider } from 'react-redux';
+import store from 'app/store';
+import { CssBaseline } from '@mui/material';
+
+const container = document.getElementById('root');
+const root = createRoot(container!);
+
+root.render(
+    <React.StrictMode>
+        <Provider store={store}>
+            <BrowserRouter>
+                <CssBaseline />
+                <App />
```

Comment:
I see this twice in your code. Is it ok?

```
<CssBaseline />
<App />
```

Reply:
Update